source/_patterns/basics/10-tags/tag.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Tag
+---
+## Overview
+
+This is a Bootstrap component, [see complete documentation here](http://v4-alpha.getbootstrap.com/components/tag/)
+
+NOTE: Bootstrap 4 renames Labels to be Tags in order disambiguate with the <label> tag. See [http://getbootstrap.com/components/#labels](http://getbootstrap.com/components/#labels) for complete labels documentation.
```

Comment:
I like the `NOTE`, lets add it to the template and make it `**Note:**` so its bold